src/tests/mod.rs

```diff
@@ -0,0 +1,248 @@
+use crate::retrieve::app::retrieve_app;
+use crate::send::app::internal_send_app;
+use crate::send::app::send_app;
+use crate::settings::AvatarSettings;
+use crate::storage::loader::filesystem::FilesystemLoader;
+use crate::storage::saver::filesystem::FilesystemSaver;
+use actix_web::dev::Body;
+use actix_web::dev::ResponseBody;
+use actix_web::dev::Service;
+use actix_web::middleware::Logger;
+use actix_web::test;
+use actix_web::web;
+use actix_web::web::Data;
+use actix_web::App;
+use actix_web::HttpMessage;
+use bytes::Bytes;
+use cis_client::getby::GetBy;
+use cis_client::AsyncCisClientTrait;
+use cis_client::CisFut;
+use cis_profile::crypto::SecretStore;
+use cis_profile::schema::Profile;
+use dino_park_gate::scope::ScopeAndUser;
+use dino_park_trust::AALevel;
+use dino_park_trust::GroupsTrust;
+use dino_park_trust::Trust;
+use failure::Error;
+use lru_time_cache::LruCache;
+use serde::Deserialize;
+use serde_json::Value;
+use std::env;
+use std::marker::Send;
+use std::sync::Arc;
+use std::sync::Mutex;
+
+#[derive(Clone)]
+pub struct MockCisClient {}
+
+unsafe impl Sync for MockCisClient {}
+
+unsafe impl Send for MockCisClient {}
+
+impl AsyncCisClientTrait for MockCisClient {
+    fn get_user_by(&self, _id: &str, _by: &GetBy, _filter: Option<&str>) -> CisFut<Profile> {
+        unimplemented!();
+    }
+    fn get_inactive_user_by(
+        &self,
+        _id: &str,
+        _by: &GetBy,
+        _filter: Option<&str>,
+    ) -> CisFut<Profile> {
+        unimplemented!();
+    }
+    fn get_any_user_by(&self, _id: &str, _by: &GetBy, _filter: Option<&str>) -> CisFut<Profile> {
+        unimplemented!();
+    }
+    fn update_user(&self, _id: &str, _profile: Profile) -> CisFut<Value> {
+        unimplemented!();
+    }
+    fn update_users(&self, _profiles: &[Profile]) -> CisFut<Value> {
+        unimplemented!();
+    }
+    fn delete_user(&self, _id: &str, _profile: Profile) -> CisFut<Value> {
+        unimplemented!();
+    }
+    fn get_secret_store(&self) -> &SecretStore {
+        unimplemented!();
+    }
+}
+
+#[actix_rt::test]
+async fn healthz_check_returns_success() -> Result<(), Error> {
+    let app = App::new()
+        .wrap(Logger::default())
+        .service(web::scope("").service(crate::healthz::healthz_app()));
+    let mut app = test::init_service(app).await;
+
+    let req = test::TestRequest::get().uri("/healthz").to_request();
+
+    let res = test::call_service(&mut app, req).await;
+    assert!(res.response().status().is_success());
+
+    Ok(())
+}
+
+#[actix_rt::test]
+#[cfg(feature = "localuserscope")]
```

Comment:
That’s not needed, is it?

Reply:
Ah, nice catch. Yeah, it is not needed anymore when we supply `--all-features`